app/controllers/attendances_controller.rb

```diff
@@ -1,5 +1,12 @@
 class AttendancesController < ApplicationController
   def index
+    date = params[:date]
+    if !date.nil?
+      @attendance = Attendance.where(attendance_date: date)
+    else
+      @attendance = Attendance.where(attendance_date: Time.now)
+    end
+    render 'index'
   end
```

Comment:
this code is a little bit hard to understand at first, you need to stop and think a little bit, and one of the nice things that ruby offer is that you can write code like this that makes easier to understand the code:

```
def index
  date = params[:date]

  @attendance = Attendance.where(attendance_date: date) unless date.nil?
  @attendance = Attendance.where(attendance_date: Time.now) if date.nil?

  render 'index'
end
```